.travis.yml

```diff
@@ -13,6 +13,7 @@ matrix:
       dist: trusty
       sudo: enabled
     - os: osx
+      osx_image: xcode7.3
```

Comment:
If this is the temporal soultion, what about adding // Fix me : description ?

Reply:
Done.